run.py

```diff
@@ -155,6 +157,17 @@ async def on_message(message):
         elif commands[0].lower() == 'recent-tweet':
             msg = helper.most_recent_tweet()
             await message.channel.send(msg)
+        elif commands[0].lower() == 'bohn':
+            try:
+                tagged = message.mentions[0].name
+                if tagged == "kalen":
+                    await message.channel.send("I will not bohn my creator")
+                    return
+                response = ("You've been bohned " + str(tagged))
+                await message.channel.send(response)
+                return
```

Comment:
Convert this to an if-else or remove the return on line 168.
```suggestion
                tagged = message.mentions[0].name
                if tagged == "kalen":
                    await message.channel.send("I will not bohn my creator")
                else:
                    response = (f"You've been bohned {tagged}")
                    await message.channel.send(response)
```

It is possible for anyone to change their name to "kalen" which will then print the secret message. If you'd like to prevent that you could check by ID (although your ID would then be in a public repo): `if message.mentions[0].id == 123456789:`. Here is how to get your ID: https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID- Up to you!